src/docfx/build/toc/TableOfContentsLoader.cs

```diff
@@ -67,13 +67,14 @@ internal class TableOfContentsLoader
                 var servicePages = new List<FilePath>();
                 var referencedFiles = new List<FilePath>();
                 var referencedTocs = new List<FilePath>();
-                var node = LoadTocFile(file, file, referencedFiles, referencedTocs);
+                TableOfContentsNode node = new TableOfContentsNode();
```

Comment:
initialization is not needed

Reply:
For the return statements in L95, it's needed.

Reply:
It is initialized either in L76 or L92

Reply:
Fixed.